src/main/cpp/kernels/SystemML.ptx

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
In recent commit, I enabled option 3 by not requiring .ptx file to be included in git. To use GPU kernels, the user will have to compile using the command `mvn package -DenableGPU=true`. This option expects that the user will have to setup nvcc on his dev environment to use GPU. To reduce the number of artifacts, we can include the GPU kernels by default in our releases ...  TODO: set `-DenableGPU=true` in dev/release scripts.